src/components/orders/OrdersUserDetailsTable/index.tsx

```diff
@@ -22,6 +23,45 @@ const Wrapper = styled(StyledUserDetailsTable)`
   > tbody > tr {
     grid-template-columns: 12rem 7rem repeat(2, minmax(16rem, 1.5fr)) repeat(2, minmax(18rem, 2fr)) 1fr;
   }
+
+  .header-title {
+    display: none;
+  }
+
+  @media ${MEDIA.mobile} {
```

Comment:
Move the media query inside the existing selector on lines 22/23

Reply:
Better:
- Turn the `table` into a styled component or import an existing table. This keeps both the `Wrapper` component and `table` selector dryer.

Reply:
> Better:
> 
> * Turn the `table` into a styled component or import an existing table. This keeps both the `Wrapper` component and `table` selector dryer.

I don't understand this. Wrapper component is already a Table styled component.

Reply:
> Move the media query inside the existing selector on lines 22/23

But the selector used in 22/23 is for common styles for thead and tr. Moving the media query there would not work the same.